### PR TITLE
get retirement info functions created

### PR DESCRIPTION
**Pull request by khasanby**

### CODE REVIEW

File: iTechArt.Repository/Repositories/RetirementRepository.cs [hunk 1]
```diff
@@ -28,32 +28,105 @@ public RetirementRepository(AppDbContext dbContext, IMapper mapper)
...
+            int Y1995to99 = retiredPolices.Where(c => c.BirthDate.Year + RetirementAgePolice >= 1995 && c.BirthDate.Year + RetirementAgePolice < 2000).ToList().Count;
+            int Y2000to04 = retiredPolices.Where(c => c.BirthDate.Year + RetirementAgePolice >= 2000 && c.BirthDate.Year + RetirementAgePolice < 2005).ToList().Count;
+            int Y2005to09 = retiredPolices.Where(c => c.BirthDate.Year + RetirementAgePolice >= 2005 && c.BirthDate.Year + RetirementAgePolice < 2010).ToList().Count;
+            int Y2010to14 = retiredPolices.Where(c => c.BirthDate.Year + RetirementAgePolice >= 2010 && c.BirthDate.Year + RetirementAgePolice < 2015).ToList().Count;
+            int Y2015to19 = retiredPolices.Where(c => c.BirthDate.Year + RetirementAgePolice >= 2015 && c.BirthDate.Year + RetirementAgePolice < 2020).ToList().Count;
+            int Y2020to21 = retiredPolices.Where(c => c.BirthDate.Year + RetirementAgePolice >= 2020 && c.BirthDate.Year + RetirementAgePolice < 2022).ToList().Count;
+            int Y2022 = retiredPolices.Where(c => c.BirthDate.Year + RetirementAgePolice == 2022).ToList().Count;
```
Comment (shurik86) [inline iTechArt.Repository/Repositories/RetirementRepository.cs]: Filter must allow user to set custom date range

File: iTechArt.Api/Controllers/GroceryController.cs [hunk 2]
```diff
@@ -12,10 +12,12 @@ public sealed class GroceryController : Controller
         private const string CSV = "csv";
         private const string EXCEL = "officedocument.spreadsheetml.sheet";
         private const string XML = "xml";
+        private readonly IGetRetirementInfo _getRetirementInfo;
 
-        public GroceryController(IGroceryService groceryService)
+        public GroceryController(IGroceryService groceryService, IGetRetirementInfo getRetirementInfo)
         {
```
Comment (shurik86) [inline iTechArt.Api/Controllers/GroceryController.cs:18]: IGetRetirementInfo - it is name for model, not for service/repository/etc

**Review by shurik86 — COMMENTED** [review]
I am waiting your changes)

File: iTechArt.Repository/Repositories/RetirementRepository.cs (hunk 1 above)
Comment (khasanby) [reply iTechArt.Repository/Repositories/RetirementRepository.cs]: It is not filter, it is for graph data

**Pull request merged**